### DISCUSSION TITLE
Change to the process of resetting tokens

### DISCUSSION THREAD

**Discussion by Bowiemtl** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#757] [Ideas]
First off I have been having the app for a while now and while I love it I do think the way the token is reset could be improved. Currently it tells me that my token is expired, relaunches, opens my browser, requests my token and relaunches again. I think that as soon as it tells me that my token is expired there should be a button to open my browser and immediately submit my token reducing it to one relaunch. I haven't taken a look at your code and I simply don't have the time to make a pull request so I don't know if there is some limitation to this I'm not seeing but I hope you consider it because this change would make the app perfect for me.

**Comment by im-coder-lg** [comment]
I think it restarts to kinda load elements and send a ping to Google Assistant's API. When authorised, it sends another ping. I think this is why it restarts as well as updates tokens on restart.
